examples/io/plot_bids.py

```diff
@@ -0,0 +1,28 @@
+"""
+===================================
+Create a BIDS-compatible MEG folder
+===================================
+
+Brain Imaging Data Structure (BIDS) MEG is a new standard for
+storing MEG files. This example demonstrates how to convert
+your existing files into a BIDS-compatible folder.
+"""
+
+# Authors: Mainak Jas <mainak.jas@telecom-paristech.fr>
+#          Alexandre Gramfort <alexandre.gramfort@telecom-paristech.fr>
+
+# License: BSD (3-clause)
+
+import os.path as op
+from mne.datasets import sample
+from mne.bids import folder_to_bids
+
+data_path = sample.data_path()
+
+fnames = dict(events='sample_audvis_raw-eve.fif',
+              raw='sample_audvis_raw.fif')
+
```

Comment:
yeah maybe eventually. For now, I'll try to provide a link to the BIDS draft ...